agent/handlers/v3/container_metadata_handler.go

```diff
@@ -65,6 +69,7 @@ func ContainerMetadataHandler(state dockerstate.TaskEngineState) func(http.Respo
 func GetContainerResponse(containerID string, state dockerstate.TaskEngineState) (*v2.ContainerResponse, error) {
 	containerResponse, err := v2.NewContainerResponse(containerID, state)
 	if err != nil {
+		seelog.Errorf("unable to get container metadata for container '%s'", containerID)
```

Comment:
nit: capitalize the start of the log message.

Reply:
Done.